src/skeleton/skeleton-text.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, Input, OnChanges } from "@angular/core";
+
+@Component({
+	selector: "ibm-skeleton-text",
+	template: `
+		<p
+			*ngFor="let width of lineWidths"
+			class="bx--skeleton__text"
+			[style.width]="width">
+		</p>
+	`
+})
+export class SkeletonText implements OnChanges {
+	@Input() lines = 5;
+
+	@Input() minLineWidth = 100;
+
+	@Input() maxLineWidth = 300;
+
+	lineWidths: Array<string>;
+
+	randoms = [0.973051493507435, 0.15334737213558558, 0.5671034553053769];
```

Comment:
That's... unusual...
Why not generate these one needed so they are really random?

Reply:
So the advantage here is that the lines can be more consistent, compare the react demo: http://react.carbondesignsystem.com/?path=/story/skeletontext--default to our demo: https://deploy-preview-952--c4ng.netlify.com/?path=/story/components-skeleton--skeleton-text

I'm not sure which is preferable, but the react one is far more consistent